src/commonmark/en/content/developer/web-api.md

```diff
@@ -10603,8 +10603,8 @@ will filter the analytics response. You can use it like this:
 
 In order to have the analytics resource generate the data in the shape
 of a ready-made table, you can provide the *tableLayout* parameter with
-true as value. Instead of generating a plain, normalized data source,
-the analytics resource will now generate the data in table layout. You
+true as a value. Instead of generating a plain, normalized data source,
```

Comment:
I think it's better with either "true as value" or "true as the value", but not "true as a value".